seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-7/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcXuguIT.java

```diff
@@ -0,0 +1,246 @@
+        String[] fieldNames = testDataSet.getKey();
+
+        String insertSql = insertTable(XUGU_SCHEMA, XUGU_SOURCE, fieldNames);
+
+        return JdbcCase.builder()
+                .dockerImage(XUGU_IMAGE)
+                .networkAliases(XUGU_CONTAINER_HOST)
+                .containerEnv(containerEnv)
+                .driverClass(DRIVER_CLASS)
+                .host(HOST)
+                .port(XUGU_PORT)
+                .localPort(XUGU_PORT)
+                .jdbcTemplate(XUGU_URL)
+                .jdbcUrl(jdbcUrl)
+                .userName(XUGU_USERNAME)
+                .password(XUGU_PASSWORD)
+                .schema(XUGU_SCHEMA)
+                .database(XUGU_DATABASE)
+                .sourceTable(XUGU_SOURCE)
+                .sinkTable(XUGU_SINK)
+                .catalogDatabase(CATALOG_DATABASE)
+                .catalogSchema(XUGU_SCHEMA)
+                .catalogTable(CATALOG_TABLE)
+                .createSql(CREATE_SQL)
+                .configFile(CONFIG_FILE)
+                .insertSql(insertSql)
+                .testData(testDataSet)
+                .build();
+    }
+
+    @Override
+    void compareResult(String executeKey) {
+        defaultCompare(executeKey, fieldNames, "XUGU_INT");
+    }
+
+    @Override
+    String driverUrl() {
+        return "https://repo1.maven.org/maven2/com/xugudb/xugu-jdbc/12.2.0/xugu-jdbc-12.2.0.jar";
+    }
+
+    @Override
+    Pair<String[], List<SeaTunnelRow>> initTestData() {
+        List<SeaTunnelRow> rows = new ArrayList<>();
+        for (int i = 0; i < 100; i++) {
+            SeaTunnelRow row =
+                    new SeaTunnelRow(
+                            new Object[] {
+                                BigDecimal.valueOf(1.12),
+                                BigDecimal.valueOf(i, 2),
+                                i,
+                                i,
+                                Long.parseLong("1"),
+                                i,
+                                i,
+                                Float.parseFloat("1.1"),
+                                Double.parseDouble("1.1"),
+                                String.format("f1_%s", i),
+                                String.format("f1_%s", i),
+                                String.format("f1_%s", i),
+                                String.format("f1_%s", i),
+                                String.format("f1_%s", i),
+                                Date.valueOf(LocalDate.now()),
+                                Time.valueOf(LocalTime.now()),
+                                new Timestamp(System.currentTimeMillis()),
+                                Timestamp.valueOf(LocalDateTime.now()),
+                                Time.valueOf(LocalTime.now()),
+                                new Timestamp(System.currentTimeMillis()),
+                                null,
+                                null,
+                                null,
+                                null
```

Comment:
bool field is null？

Reply:
yes，The Boolean data type has three states: true, false, and unknown. Field values can be true, false, null